proposals/0258-property-wrappers.md

```diff
@@ -679,15 +679,15 @@ in one of three ways:
 ### Type inference with wrappers
 
 Type inference for properties with wrappers involves both the type
-annotation of the original property (if present) and the wrapper
-type, using the initialization of the synthesized stored property. For
-example:
+annotation of the original property (if present), the wrapper
+type, and the initial value (if present). The complete wrapper type is inferred, and it's `value` property is used to provide the type for the original property. For example, providing an initial value to the original property uses the `initialValue` initializer to infer the wrapper type:
```

Comment:
"its", not "it's"

Reply:
There are (at least) two senses of "type inference" that this section could be about: inferring the value type of the variable when there's no type annotation and inferring generic arguments for generic wrapper types.  Could you reword the start to be clearer about what you mean here, like "When a wrapped property lacks an explicit type annotation, its value type is inferred considering information from both the wrapper type's `value` property and the initializer expression.  This value type is then used to infer generic arguments for the wrapper type if it is generic and those arguments weren't given explicitly."

Also, I don't see anything here explaining how this rule interacts with composition.

Reply:
This section is about both of those... I'll try to generalize the text a little.